### PR TITLE
SLEM PC Update with transactional register

### PR DESCRIPTION
**Pull request by m-dati**
SLEM PC update: add SCC registering via transactional update

- Related ticket: https://progress.opensuse.org/issues/137168
- Needles: N/A
- Verification run: PASSED
  - sle-micro-5.4-GCE-BYOS-Updates-x86_64-Build20240325-1-slem_basic@64bit 
    https://openqa.suse.de/tests/13871694
  - sle-micro-5.3-GCE-BYOS-Updates-x86_64-Build20240325-1-slem_basic@64bit    
    https://openqa.suse.de/tests/13871676

### CODE REVIEW

File: lib/transactional.pm [hunk 1]
```diff
@@ -205,6 +205,8 @@ sub trup_call {
     $args{timeout} //= 180;
     $args{exit_code} //= 0;
     $args{proceed_on_failure} //= 0;
+    $args{sudo} //= 0;
+    $script = "sudo " . $script if ($args{sudo});
```
Comment (ricardobranco777) [inline lib/transactional.pm:209]: Why is this needed?

Comment (m-dati) [reply lib/transactional.pm:209]: Ah leftover of a previous version attempt

**Review by ricardobranco777 — APPROVED** [review]

File: lib/transactional.pm (hunk 1 above)
Comment (m-dati) [reply lib/transactional.pm]: Removed.
